layouts/partials/labs.html

```diff
@@ -0,0 +1,80 @@
+<section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+    <h2>Lab 1 - Introduction to Falco</h2>
+    <p>General introduction to Falco in single Linux hosts. Learn how to install Falco with Docker, and detect threads
+      like:</p>
+    <ul>
+      <li>Container running interactive shell</li>
+      <li>Unauthorized process</li>
+      <li>Write to non user-data directory</li>
+    </ul>
+    <p>This lab is the best place to start your Runtime Security jurney!</p>
+
+    <div style="text-align: left">
+      <a href="/labs/lab1">
+        <button style='background-color:#00aec7;margin-top:1px;margin-bottom:1px;border-radius:4px;padding:8px;
+          font-family: "Open Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    font-size: 14px; color:#FFFFFFBF; border-width: 0px;'>
+          Launch lab1 environment!
+        </button>
+      </a>
+    </div>
+
+  </div>
+
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+    <h2>Lab 2 - Falco forensics in K8s</h2>
+    <p>In this lab, you can learn how to deploy Falco in a a running Kubernetes cluster and how to define your first
+      custom rules file.</p>
+    <p>Play the both the intruder & sysadmin roles and detect a Rogue attack</p>
+
+    <div style="text-align: left">
+      <a href="/labs/lab2">
+        <button style='background-color:#00aec7;margin-top:1px;margin-bottom:1px;border-radius:4px;padding:8px;
+          font-family: "Open Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+    font-size: 14px; color:#FFFFFFBF; border-width: 0px;'>
+          Launch lab2 environment!
+        </button>
+      </a>
+    </div>
+
+  </div>
+</section>
+
+<section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+    <h2>Lab 3 - Falcosidekick: Thread Response Engine</h2>
+    <p>Falco is not just about Runtime Detection, you can perform automated actions to mitigate threads automatically.
+    </p>
+    <p>In this lab you will learn how to deploy Falco with Falcosidekick and Kubeless. Stop a running pod when you
```

Comment:
In this lab, you will learn how to deploy Falco with Falcosidekick and Kubeless. Stop the running pod when you detect a security thread!